File: 2-Decisao/13.rb
#!/usr/bin/env ruby

semana = {1 => 'Domingo', 2 => 'Segunda-Feira', 3 => 'Terça-Feira', 4 => 'Quarta-Feira', 5 => 'Quinta-Feira', 6 => 'Sexta-Feira', 7 => 'Sabado'}

dia = gets.chomp.to_i

if semana.key? dia
    puts semana[dia]
else
    puts 'Valor Invalido'
end

File: 2-Decisao/13.py
#!/usr/bin/env python
#coding:utf-8
semana = {1: 'Domingo', 2: 'Segunda-Feira', 3: 'Terça-Feira', 4: 'Quarta-Feira', 5: 'Quinta-Feira', 6: 'Sexta-Feira', 7: 'Sabado'}

dia = input("")

if semana.has_key(dia):
    print semana[dia]
else:
    print 'Valor Invalido'